### DISCUSSION TITLE
Hide menu to display “home” screen until I summon the menu?

### DISCUSSION THREAD

**Discussion by NWSpitfire** [forntoh/LcdMenu#272] [General]
Hello,

I am building a current source project which will require a menu for my 20x04 LCD and I am interested in using this library but I am a little confused.

My hope is to have a standard Home Screen, that is displayed by default and shows current temperature, current and a few other things. But then when I press the rotary encoder it will summon the menu which will contain sub menus and commands to run and save PID, set target current etc. 

Is this possible? As I understand it the main menu is what is displayed at all times and you can scroll to sub menus to execute commands? Or can I have a custom screen as default and only summon the menu when required?

I have seen the MenuTimeout function, does this allow the menu to time out after a set time. Is this just to disable the display for power savings or can it be used to revert to the standard home page (so exit the menus) after a certain time?

Thank you

**Comment by B1ackt34** [comment]
You should add a function based on millis ... when you dont use menu after a certain time it will hide menu and post the infos until you press button.
Take care if you have more functions running in the background and dont stop them
